docs/monitors/elasticsearch.md

```diff
@@ -0,0 +1,124 @@
+<!--- GENERATED BY gomplate from scripts/docs/monitor-page.md.tmpl --->
+
+# elasticsearch
+
+Monitors ElasticSearch instances.
```

Comment:
This needs to be a bit more descriptive.  Maybe talk about all the subtleties we have learned about master nodes and index stats, etc.

Reply:
Added more details and examples